src/linagora.esn.calendar/app/event-view/external-user/event-view-external-user.controller.js

```diff
@@ -36,8 +36,9 @@ const _ = require('lodash');
 
       $http({ method: 'GET', url: self.linksMapping[partstat] }).then(function() {
         var participation = partstat.charAt(0).toUpperCase() + partstat.slice(1).toLowerCase();
```

Comment:
```suggestion
        const participation = partstat.charAt(0).toUpperCase() + partstat.slice(1).toLowerCase();
```

Reply:
Though it is not part of the PR but you can change it to `const` to avoid mixing `var` & `const`.